game.py

```diff
@@ -20,7 +20,36 @@ def snowman(snowman_word):
     If the player wins and, 
     'Sorry, you lose! The word was {snowman_word}' if the player loses
     """
-    pass
+    correct_letter_guess_statuses = build_letter_status_dict(snowman_word)
+    wrong_guesses_list = []
+
+    # Main game loop
+    while len(wrong_guesses_list) < SNOWMAN_MAX_WRONG_GUESSES:
+
+        # Check if the word has already been fully guessed.
+        if is_word_guessed(snowman_word, correct_letter_guess_statuses):
+            print("Congratulations! You win!")
+            break
```

Comment:
What's another way the logic in this loop could be written so that you don't need to use `break` to end the game?